game/opengl_text.cc

```diff
@@ -246,22 +171,25 @@ void SvgTxtTheme::draw(Window& window, std::string _text) {
 
 void SvgTxtTheme::draw(Window& window, std::vector<TZoomText>& _text, bool lyrics) {
 	std::string tmp;
-	for (auto& zt: _text) { tmp += zt.string; }
+
+	for (auto& zt: _text)
+		tmp += zt.string;
```

Comment:
Please add the braces even though its a single statement in the forloop. This way i don't have to scan the code for more indentations (that's the reason it's been single lined i guess) to see where the loop ends.